### DISCUSSION TITLE
Why the deprecation notice?

### DISCUSSION THREAD

**Discussion by aadrian** [michelp/pgsodium#110] [Q&A]
Any idea why this notice?  https://supabase.com/docs/guides/database/extensions/pgsodium

Thank you.

**Comment by RomainFranceschini** [comment]
AFAIK, Supabase uses pgsodium in the Vault Postgres extension, and the underlying implementation will move away from pgsodium to avoid storing keys in PG.

However, I'm interested in using pgsodium TCE feature.

Does this annoucement means the project will be completely abandoned? Or in maintenance mode?
